src/btree/bt_cursor.c

```diff
@@ -612,6 +612,8 @@ __wt_btcur_reset(WT_CURSOR_BTREE *cbt)
     WT_STAT_CONN_DATA_INCR(session, cursor_reset);
 
     F_CLR(cursor, WT_CURSTD_KEY_SET | WT_CURSTD_VALUE_SET);
+    cbt->upd_value->type = WT_UPDATE_INVALID;
+    WT_TIME_WINDOW_INIT(&cbt->upd_value->tw);
```

Comment:
The code is OK to not clear it. There is no correctness issue. I did this because I see in the core dump that we have garbage value for the values so I think better to fix that to not confuse us when debugging.